File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/lexer/Scala3LexerTest.java
package org.jetbrains.plugins.scala.lang.lexer;

import junit.framework.Test;
import junit.framework.TestCase;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.plugins.scala.Scala3Language;

public class Scala3LexerTest extends TestCase {
    @NotNull
    public static Test suite() {
        return new ScalaLexerTestBase("/lexer/data3") {
            @NotNull
            @Override
            protected Scala3Language getLanguage() {
                return Scala3Language.INSTANCE;
            }
        };
    }
}

File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/lexer/Scala3LexerTest.scala
package org.jetbrains.plugins.scala.lang.lexer

import com.intellij.lang.Language
import org.jetbrains.plugins.scala.Scala3Language

import java.nio.file.Path

class Scala3LexerTest extends ScalaLexerTestBase {
  override protected def relativeTestDataPath: Path = Path.of("lexer", "data3")

  override protected def language: Language = Scala3Language.INSTANCE
}
